README.md

```diff
@@ -8,11 +8,18 @@ The recommended way to install `kallax` is:
 
 ```
 go get -u github.com/src-d/kallax/...
+go install github.com/src-d/go-kallax/generator/cli/kallax
 ```
 
 > *kallax* includes a binary tool used by [go generate](http://blog.golang.org/generate),
 please be sure that `$GOPATH/bin` is on your `$PATH`
 
+use
+---
+```
+kallax gen -i=PATH_TO_THE_PACKAGE
```

Comment:
nope, this is not going to be used like that. is meant to be used with the `//go:generate` command